src/components/molecules/ModalConfirmWithNamespaceSelect/ModalConfirmWithNamespaceSelect.tsx

```diff
@@ -115,6 +115,9 @@ const ModalConfirmWithNamespaceSelect: React.FC<IProps> = props => {
     }
   }, [defaultOption, defaultNamespace, namespaces]);
 
+  const onlyNamespaces = useMemo(() => !resources.some(r => r.kind !== 'Namespace'), [resources]);
```

Comment:
This one is easier to read if you write it as:
`resources.every(r => r.kind === 'Namespace')`

Reply:
ha! i didn't know of `every` - thanks!

Reply:
@devcatalin fixed